### PR TITLE
Add shadow fork configs

### PR DESCRIPTION
**Pull request by parithosh**
Adds:
- Small sepolia shadow fork config
- Bigger mainnet shadow fork config
- Updates playbooks to support shadow forks
- Fixes validator check by filtering it through bool

### CODE REVIEW

File: withdrawal-devnet-2/inventory/group_vars/teku.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ beacon_start_args: >
   --log-destination=console
   --rest-api-host-allowlist="*"
   --data-storage-non-canonical-blocks-enabled=true
-  {% if no_validator_keys != 'true' %}
+  {% if no_validator_keys|bool != true %}
```
Comment (barnabasbusa) [inline withdrawal-devnet-2/inventory/group_vars/teku.yml:56]: could you rebase master, and do this change for devnet3 too?

File: secrets/ingress_users.enc.yaml [hunk 2]
```diff
@@ -1,117 +0,0 @@
-ingress_users:
```
Comment (barnabasbusa) [inline secrets/ingress_users.enc.yaml]: how come this got removed?

File: withdrawal-mainnet-shadowfork-1/inventory/group_vars/explorer.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+explorer_docker_network_name: explorernet
+
+beacon_explorer_container_name: explorer
+execution_explorer_container_name: blockscout
+
+explorer_memory: 9000m
+beacon_explorer_image_name: "parithoshj/beacon-explorer:print-chain-id"
+execution_explorer_image_name: "skylenet/blockscout:v4.0.0-beta"
```
Comment (barnabasbusa) [inline withdrawal-mainnet-shadowfork-1/inventory/group_vars/explorer.yml:8]: there are some newer images.

File: secrets/ingress_users.enc.yaml (hunk 2 above)
Comment (parithosh) [reply secrets/ingress_users.enc.yaml]: maybe i made a mistake there? unsure

File: withdrawal-mainnet-shadowfork-1/inventory/group_vars/explorer.yml (hunk 3 above)
Comment (parithosh) [reply withdrawal-mainnet-shadowfork-1/inventory/group_vars/explorer.yml:8]: don't have it deployed, update during deployment :P

File: secrets/ingress_users.enc.yaml (hunk 2 above)
Comment (barnabasbusa) [reply secrets/ingress_users.enc.yaml]: without this, you won't be able to push to grafana.

**Review by barnabasbusa — APPROVED** [review]

**Pull request merged**